rosbag2/include/rosbag2/storage_options.hpp

```diff
@@ -25,6 +25,7 @@ struct StorageOptions
 public:
   std::string uri;
   std::string storage_id;
+  uint64_t max_bagfile_size = UINT64_MAX;
```

Comment:
https://github.com/isocpp/CppCoreGuidelines/blob/master/CppCoreGuidelines.md#Rc-in-class-initializer

Reply:
By using default initialization, I believe this is not POD anymore, the other files should have defaults too.